tests/support/case.py

```diff
@@ -808,11 +821,32 @@ def minion_run(self, _function, *args, **kw):
         '''
         return self.run_function(_function, args, **kw)
 
-    def run_function(self, function, arg=(), minion_tgt='minion', timeout=300, **kwargs):
+    def run_function(self, function, arg=(), minion_tgt='minion', timeout=300, rem=False, **kwargs):
         '''
         Run a single salt function and condition the return down to match the
         behavior of the raw function call
         '''
+
+        # This not rem and minion_tgt block changes how most integration
+        # tests are run. Instead of running client -> master -> minion -> os
+        # and then back again, it skips all of that by just running the
+        # function from the MasterMinion.  This should work in most cases,
+        # but for those that really do need the round trip, or only make
+        # sense when executed in the context of a minion, `rem=True` (where
+        # rem means remote), allows bypassing this. It incurs the cost of
```

Comment:
And why not `remote` instead of `rem`?
Saves a read on the comment....

Reply:
🤷‍♂ I don't know if I have an *strong* feelings. `rem` was just what Tom suggested while we were working :) I *do* kind of like it being terse and somewhat cryptic as an indication that you probably shouldn't be using it unless you understand why you're using it. But if there are strong feelings the other way, I'm happy to change it.

Reply:
I thought that name would have come from @thatch45 or a similar mind :)

He like short names, I like understanding the names as I read them.
Your cryptic argument sold me.
Leave it.